Makefile

```diff
@@ -19,3 +19,12 @@ lint: prometheus_alert_rules.yaml
 
 clean:
 	rm -rf prometheus_alert_rules.yaml
+
+generate_k8s:
+	cd extras; cp manifests/prometheus-rules.yaml .; jb install; ./build.sh example.jsonnet; cd -
```

Comment:
instead of semicolon(`;`) it's better we use `&&` syntax.

Reply:
I understand both have different purpose. With && if previous one is successful then only second would be executed. Anyway let me try that.